### DISCUSSION TITLE
Null check operator used on a null value on  DatabaseConnection._formatTableRow

### DISCUSSION THREAD

**Discussion by laldevfr** [serverpod/serverpod#296] [Q&A] [answered]
Hi,

I try to use serverpod, but i face to a issue
When i manipulated database, the server return a error with Null Check operator : 
`
Null check operator used on a null value
#0      DatabaseConnection._formatTableRow (package:serverpod/src/database/database_connection.dart:223:34)
#1      DatabaseConnection.find (package:serverpod/src/database/database_connection.dart:178:18)
<asynchronous suspension>
#2      Database.find (package:serverpod/src/database/database.dart:60:12)
<asynchronous suspension>
#3      TodoEndpoint.getTodo (package:todoapp_server/src/endpoints/todo_endpoint.dart:10:29)
<asynchronous suspension>
#4      EndpointDispatch.handleUriCall (package:serverpod/src/server/endpoint_dispatch.dart:116:20)
<asynchronous suspension>
#5      Server._handleRequest (package:serverpod/src/server/server.dart:221:18)
<asynchronous suspension>
`
version : serverpod 9.11 on MacOs.

**Comment by vlidholt** [comment, marked as answer]
Serverpod doesn't automatically keep the database up-to-date with the yaml files (yet). So whichever tables you add to the yaml files, you will also need to add to your database.
